src/style.css

```
@@ -0,0 +1,137 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 14px;
+  box-sizing: border-box;
+
+  --background-color: #fff;
+  --border-color: #e1e7ed;
+}
+
+body {
+  box-sizing: border-box;
+}
+
+/* #region form */
+.big__form {
+  position: relative;
+}
+
+.small__form {
+  position: relative;
+}
+/* #endregion */
+
+/* #region image */
+.image {
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 26px;
+  height: 19px;
+  color: #687c94;
+}
+
+.big__image {
+  top: 25px;
+  left: 26px;
+}
+
+.small__image {
+  top: 15px;
+  left: 13px;
+  height: 11px;
+  width: 11px;
+}
```

If you reset these   top: 25px; left: 26px; values for big and small images anyway so no needs to set it to common class. Check simialr cases
```suggestion
.image {
  margin: 0;
  position: absolute;
  height: 19px;
  color: #687c94;
}

.big__image {
  top: 25px;
  left: 26px;
}

.small__image {
  top: 15px;
  left: 13px;
  height: 11px;
  width: 11px;
}
```